systemtests/src/main/java/io/enmasse/systemtest/framework/TestPlanInfo.java

```diff
@@ -131,12 +132,12 @@ private boolean evaluateTestDisabled(ExtensionContext context, ExecutionConditio
 
     public void printTestClasses() {
         if (testClasses != null) {
-            LOGGER.info("****************************************************************************");
-            LOGGER.info("                 Following test classes will run                            ");
-            LOGGER.info("****************************************************************************");
+            LOGGER.info(String.join("", Collections.nCopies(90, "*")));
+            LOGGER.info("                     Following test classes will run");
+            LOGGER.info(String.join("", Collections.nCopies(90, "*")));
```

Comment:
Maybe make this string a constant somewhere in case we can to change it! (I.e. static testSeparator)